packages/boxel/.eslintrc.js

```diff
@@ -56,7 +57,7 @@ module.exports = {
         {},
         require('eslint-plugin-node').configs.recommended.rules,
         {
-          // add your custom rules and overrides for node files here
+          'node/no-unpublished-require': 'off',
```

Comment:
Could this be applied more narrowly?

Reply:
Probably, but we disable this in our other packages, so the consistency seems like a good move.